xml/System.ServiceModel/WSDualHttpBinding.xml

```diff
@@ -711,7 +711,7 @@
    
   
 ## Examples  
- The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System.ServiceModel.Channels.IBindingRuntimePreferences.ReceiveSynchronously%2A> value.  
+ The following examples show how to get the <xref:System.ServiceModel.WSDualHttpBinding.System%23ServiceModel%23Channels%23IBindingRuntimePreferences%23ReceiveSynchronously%2A> value.  
```

Comment:
I'm thinking that the broken link should actually go to WSDualHttpBinding, not BasicHttpBinding, no?